server/neptune/workflows/internal/activities/execute.go

```diff
@@ -1,12 +1,99 @@
 package activities
 
-import "context"
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
 
-type executeCommandActivities struct{}
+	"github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/runtime/cache"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/steps"
+)
+
+type executeCommandActivities struct {
+	// Version Cache ensures the terraform version exists
+	VersionCache cache.ExecutionVersionCache
+
+	// DefaultTFVersion is the default configured version for atlantis is the tfVersion is not supplied
+	DefaultTFVersion *version.Version
+
+	// TerraformBinDir is the directory where Atlantis downloads Terraform binaries.
+	TerraformBinDir string
+}
+
+type ExecuteCommandResponse struct {
+	Output string
+	Error  error
+}
 
 type ExecuteCommandRequest struct {
+	Step           steps.Step
+	Path           string
+	CustomEnvVars  map[string]string
+	DefaultEnvVars map[string]string
+
```

Comment:
I had it setup like this since the the woker would only be able to assert the terraform version. But, since we don't need the terraform version anymore, I have consolidated the env vars into a single attribute.